packages/workspace/src/migrations/update-10-3-0/update-10-3-0.ts

```diff
@@ -0,0 +1,11 @@
+import { formatFiles, updatePackagesInPackageJson } from '@nrwl/workspace';
+import { join } from 'path';
+import { chain } from '@angular-devkit/schematics';
+
+const updatePrettierPackage = updatePackagesInPackageJson(
```

Comment:
Can you rename it to updatePackages? Other packages can be updated there as well